tools/data/create_subset.py

```diff
@@ -0,0 +1,112 @@
+import shutil
+import configargparse
+import random
+from pathlib import Path
+from satellitepy.utils.path_utils import create_folder
+
+def get_args():
+    """Arguments parser."""
+    parser = configargparse.ArgumentParser(description=__doc__)
+    parser.add_argument('--in-image-folder', type=Path, required=True,
+                        help='Folder of original images. The images in this folder will be processed.')
+    parser.add_argument('--in-label-folder', type=Path, required=True,
+                        help='Folder of original labels. The labels in this folder will be used to create patches.')
+    parser.add_argument('--in-mask-folder', type=Path, required=False,
+                        help='Folder of original masks. The masks in this folder will be processed.')
+    parser.add_argument('--train-out-folder', type=Path, required=True,
+                        help='Save folder of train set. Images and corresponding labels will be saved into '
+                             '<out-folder>/images and <out-folder>/labels.')
+    parser.add_argument('--val-out-folder', type=Path, required=True,
+                        help='Save folder of val set. Images and corresponding labels will be saved into '
+                             '<out-folder>/images and <out-folder>/labels.')
+    parser.add_argument('--test-out-folder', type=Path, required=True,
+                        help='Save folder of test set. Images and corresponding labels will be saved into '
+                             '<out-folder>/images and <out-folder>/labels.')
+    parser.add_argument('--randomise', type=bool, required=False, default=False, 
+                        help='Determines if the files should be randomised before copying them to new locations. Default=False')
+    parser.add_argument('--val-split', type=float, required=False, default=0.1,
+                        help='Which percantage of the split should be used for val. Default=0.1')
+    parser.add_argument('--test-split', type=float, required=False, default=0.2,
+                        help='Which percantage of the split should be used for test. Default=0.2')
+    parser.add_argument('--train-split', type=float, required=False, default=0.7,
+                        help='Which percantage of the split should be used for train. Default=0.7')
+    return parser
+
+def copy_files(files, src_folder, dst_folder):
+    for file in files:
+        print(f'Copying {src_folder / file.name} to {dst_folder / file.name}')
+        shutil.copy(src_folder / file.name, dst_folder / file.name)
+
+def run(parser):
+    args = parser.parse_args()
+
+    if args.val_split + args.test_split + args.train_split != 1:
+        print('Percentages of val, test and train splits are not equal to 1!')
```

Comment:
Use the logger instead of print; See other scripts for comparison. Not needed here either way, so just a reminder.